build.gradle

```diff
@@ -5,13 +5,16 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.5.1'
     }
 }
 
 allprojects {
     repositories {
         jcenter()
         maven { url 'https://jitpack.io' }
+        // Android studio requires google m2 to obtain support libraries, even though
+        // we are not directly referencing them at present.
+        google()
```

Comment:
...but we really should use the support libraries, or the new JetPack libraries.

https://developer.android.com/jetpack